LiteCore/Storage/SQLiteDataFile.cc

```diff
@@ -547,7 +547,8 @@ namespace litecore {
             
             // Create a SQLite view of a union of both stores, for use in queries:
 #define COLUMNS "key,sequence,flags,version,body,extra,expiration"
-            const char *cname = name.c_str();
+            string tableName = keyStore->tableName().substr(3); // remove prefix "kv_"
```

Comment:
Fragile.  Would probably be better to use a static integer here based on the length of the prefix, rather than the magic number 3.

Reply:
Add comments. Currently, "kv_" is used literally everywhere it's used. Agreed, we'd better avoid the use of magic "kv_v".